src/saveload.c

```diff
@@ -528,13 +528,26 @@ static void RemoveSaveIndex( Bag bag)
   LINK_BAG(bag) = bag;
 }
 
+
+
+static Char * GetKernelDescription( void )
+{
+    static Char SyKernelDescription[256];
+    strcpy(SyKernelDescription, SyKernelVersion);
+    if(SyUseReadline)
```

Comment:
clang-format? add issue labels?

Perhaps we can setup a bot to point this out automatically, it's a bit tiresome. Alternatively, we can also agree to not bother about clang-format anymore, and just let everybody format whichever way they feel like?

Reply:
Id be happy with a bot. Maybe i can write a git script to run on my own machine that clangformats bevore each commit. (I actually do use formst on save on other projects i work on, but that reformats the whole file, so is no good for gap).

Reply:
Now reformatted